src/Tests/Microsoft.NET.TestFramework/TestContext.cs

```diff
@@ -59,6 +59,9 @@ public void AddTestEnvironmentVariables(SdkCommandSpec command)
             command.Environment["GenerateResourceMSBuildArchitecture"] = "CurrentArchitecture";
             command.Environment["GenerateResourceMSBuildRuntime"] = "CurrentRuntime";
 
+            //  Prevent test MSBuild nodes from persisting
+            command.Environment["MSBUILDDISABLENODEREUSE"] = "1";
```

Comment:
I've been finding test runs fast in VS lately. And I suspect this may have helped a lot. But I think this is the right thing to do.

Reply:
Yeah, I think so--slower tests but hopefully fewer mysterious undiagnosable failures.

Reply:
Fair trade

Reply:
We could consider re-enabling node reuse but running `dotnet buildserver shutdown` after a test run